release/models/mpls/openconfig-mpls-igp.yang

```diff
@@ -21,7 +21,13 @@ submodule openconfig-mpls-igp {
     "Configuration generic configuration parameters for IGP-congruent
     LSPs";
 
-  oc-ext:openconfig-version "3.3.2";
+  oc-ext:openconfig-version "3.3.3";
```

Comment:
```suggestion
  oc-ext:openconfig-version "3.4.0";
```

Reply:
Here and throughout this review, the version should increment the minor version because a leaf is being added.

Reply:
Changed.